src/finufft.cpp

```diff
@@ -724,6 +723,13 @@ int FINUFFT_MAKEPLAN(int type, int dim, BIGINT* n_modes, int iflag,
     // fftw_plan_many_dft args: rank, gridsize/dim, howmany, in, inembed, istride, idist, ot, onembed, ostride, odist, sign, flags 
     {
       std::lock_guard<std::mutex> lock(fftw_lock);
+
+      // FFTW_PLAN_TH sets all future fftw_plan calls to use nthr_fft threads.
+      // FIXME: Since this might override what the user wants for fftw, we'd like to set it
+      // just for our one plan and then revert to the user value. Unfortunately
+      // fftw_planner_nthreads wasn't introduced until fftw 3.3.9, and there isn't a convenient
```

Comment:
Thanks for the clear explanation. I will add to the docs that fftw >= 3.3.6 is supported (we don't say).

Reply:
And, yeah, I failed to get fftw to return to it's original n_threads in the distant past (2017).